Issue #92 suggested solution (#93)
fredrikhgrelland/vagrant-hashistack-template: CHANGELOG.md

# Changelog

## [0.1.1 UNRELEASED]

### Added
- Colour Scheme as ad hoc variables instead of inline ad lib. 

### Changed

> - Colours in 'make template_init' termplate text output 

Reviewer: ```suggestion
### Changed
- Moved colour definitions of ‘template_init’ recipe into variables. [no issue]
```

### Fixed

> - bug where 'make template_inti' would crash on error #2 EOF.

Reviewer: ```suggestion
- bug where 'make template_init' would crash on error #2 EOF.
```

> - bug where 'make template_inti' would crash on error #2 EOF.

Reviewer: What is `#2` in this case?

- Moved colour definitions of ‘template_init’ recipe into variables. [no issue]

### Fixed

> - bug where 'make template_init' would crash on End of File error. [Issue #92]

Reviewer: ```suggestion
- bug where 'make template_init' would crash on End of File error. [#92](https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/92)
```
